fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -49,15 +49,18 @@ def load_data(self, X, y, event=None, **kwargs):
         self.index = signed.index
         self.default_objective_lambda_vec = None
 
-        # fill in the information about the basis
         event_vals = self.tags[_EVENT].unique()
         group_vals = self.tags[_GROUP_ID].unique()
+        # The matrices pos_basis and neg_basis contain a lower-dimensional description of
```

Comment:
Did you mean to include this file?

Reply:
OK, seeing that the tests make use of the moments, I can see why you might be commenting here.